### ISSUE TITLE
Feature request: sizing utility to help build apps for different screen sizes

### ISSUE THREAD

**Issue by krivachy** [futurice/pepperoni-app-kit#32]
It would be super nice to have some utility function that would help with handling UI element sizes for different screen sizes.

**Comment by jevakallio** [comment]
Agreed! Do you mean similar to the vh, vw, px stuff we have done in projects?

**Comment by krivachy** [comment]
Yes, just had to implement the same thing in a new project so I thought I might throw in an issue so we remember it.

**Comment by jevakallio** [comment]
Yeah, agreed :+1:

I would have already put that in there, but the downside of the relative screen sizes is that they don't take orientation changes into account. I would like to have a solution that doesn't make adaptive landscape/portrait apps needlessly difficult to develop.
